e2e/test/helpers/OutgoingMessageHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.Devices.E2ETests.Helpers
+{
+    internal static class OutgoingMessageHelper
+    {
+        public static OutgoingMessage ComposeOutgoingTestMessage(out string payload, out string p1Value)
```

Comment:
Moving these "compose" helpers to their own class so that they can be used by different classes more easily (more easily discoverable).